scripts/gen-i18n.js

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env node
```

Comment:
Copy pasta from https://github.com/matrix-org/matrix-react-sdk/blob/develop/scripts/gen-i18n.js

Reply:
might be beneficial to extract this to a shared place somewhere given that element-web & react-sdk already share react-sdk's feels a bit weird to have a 2nd copy of it and it might miss updates we end up doing to the other one.

Reply:
Sounds great, is there such a place that already exists for common scripts between repos?
If not I will figure out an appropriate place

Reply:
They seem to get broken up into small repos, e.g https://github.com/matrix-org/eslint-plugin-matrix-org